ibis/expr/operations.py

```diff
@@ -2883,3 +2883,118 @@ def output_type(self):
 
     def root_tables(self):
         return distinct_roots(*self.values)
+
+
+# ----------------------------------------------------------------------
+# GeoSpatial operations
+
+class GeoSpatialBinOp(BinaryOp):
+    """Geo Spatial base binary"""
+    left = Arg(rlz.geospatial)
+    right = Arg(rlz.geospatial)
+    output_type = rlz.shape_like('args', dt.float64)
```

Comment:
nit: `GeoSpatialBinOp` should not define the return type just the arguments, same for `GeoSpatialUnOp`.